components/log-viewer-webui/client/src/components/Layout/MainLayout.module.css

```
@@ -12,5 +12,6 @@
 .siderLogo {
   max-width: 100%;
   max-height: 100%;
-  height: 32px;
+  padding-inline: 10%;
```

```suggestion
  padding-inline: 15%;
```

Kinda looks better to me.